client/src/components/ImageList.jsx

```diff
@@ -0,0 +1,267 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import NextSVG from '../icons/next.svg';
+import PreviousSVG from '../icons/previous.svg';
+import Xmark from '../icons/xmark.svg';
+import SaveSVG from '../icons/save.svg';
+import ShareSVG from '../icons/share.svg';
+
+const Wrapper = styled.div`
+  position: relative;
+  height: 100%;
+  max-height: 100%;
+  width: 100vw;
+  max-width: 100vw;
+  display: flex;
+  flex-direction: column;
+`;
+
+const HeaderWrapper = styled.section`
+  display:block;
+ `;
+
+const Header = styled.div`
+  padding: 40px 40px 40px 20px;
+  display: flex;
+  align-items: center;
+  z-index: 1;
+  border-sizing: border-box;
+  flex-grow: 0;
+  flex-shrink: 0;
+  flex-basis: auto;
+`;
+
+const CloseButtonContainer = styled.div`
+  flex: 1;
+  justify-content: space-between;
+  min-width: 50px;
+  flex-grow: 0;
+  flex-shrink: 0;
+  flex-basis: auto;
+  display: block;
+
+`;
+
+const CloseButton = styled.button`
+  cursor: pointer;
+  display: incline-block
+  border-sizing: border-box;
```

Comment:
Is this supposed to be box-sizing or border-sizing?

Reply:
Thanks. It is box-sizing.